### PR TITLE
Added total document count to footer matching total database count format

### PR DESCRIPTION
**Pull request by iamgollum**
See discussion in #998 

Number 3 on List: `On the all-documents page there is no total documents count`

### CODE REVIEW

**Comment by iamgollum** [comment]
I am using VSCode and I notice the format is not matching...need to look into this

File: app/addons/documents/index-results/components/pagination/PaginationFooter.js [hunk 1]
```diff
@@ -10,50 +10,70 @@
...
     const { docs, pageStart, pageEnd } = this.props;
 
     if (docs.length === 0) {
       return <span>Showing 0 documents.</span>;
     }
 
-    return <span>Showing document {pageStart} - {pageEnd}.</span>;
+    return (
+      <span>
+        Showing document {pageStart} - {pageEnd} of {docs.length} documents.
```
[the remark this answers is not in the document]
Comment (iamgollum) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: @Antonio-Maranhao than you for the transparency here and I like the alternatives you presented. Ill play around with it.

[the remark this answers is not in the document]
Comment (iamgollum) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: @Antonio-Maranhao does this project have a Kanban board some history of User Stories?

[the remark this answers is not in the document]
Comment (Antonio-Maranhao) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: @iamgollum Not that I'm aware of.
The rationale behind code changes are AFAIK documented directly on the PRs/Issues.

[the remark this answers is not in the document]
Comment (garrensmith) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: THere are issues in this project  you can look at.

[the remark this answers is not in the document]
Comment (garrensmith) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: I'm guessing you have prettier or something in vscode autoformatting your code on save. That is causing all the formatting changes here.

**Comment by garrensmith** [comment]
As @Antonio-Maranhao says this is very difficult to do. If you have an index (_all_docs, mango or map/reduce) with 1000 documents in it. Then you do a query that returns 20 docs. The total_rows will always say 1000 docs. Its the count of the number of documents in the index not the number of documents returned. Then for mango it gets more complicated because we could technically get 100 docs from an index but then we filter it in memory to make sure docs match the selector. 

Total rows is always the number of documents in the index.

**Comment by iamgollum** [comment]
@garrensmith I feel like that should be documented somewhere - no?

File: app/addons/documents/index-results/components/pagination/PaginationFooter.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (iamgollum) [reply app/addons/documents/index-results/components/pagination/PaginationFooter.js]: @garrensmith is there a VSCode setup you guys recommend. Most projects usually have an editorconfig, prettier, etc... configured to standardize the formatting. Even when I disable prettier, some files like `wrappedAutocomplete` with `{value: el, label: el};` ... will add spaces between the brackets.

**Pull request closed without merging**